src/main/java/org/sagebionetworks/web/client/presenter/QuizPresenter.java

```diff
@@ -227,8 +237,12 @@ public String mayStop() {
 
 	@Override
 	public void setPlace(org.sagebionetworks.web.client.place.Quiz place) {
-		view.setPresenter(this);
-		view.clear();
+		if(DisplayUtils.isInTestWebsite(ginInjector.getCookieProvider())){
+			this.place = place;
```

Comment:
Is storing the place necessary?  If so, safe to always store it (not just in alpha mode)